src/Data/CachedContent.php

```
@@ -0,0 +1,54 @@
+        public readonly string $createTime,
+        public readonly string $updateTime,
+        public readonly string $expireTime,
+    ) {}
+
+    /**
+     * @param  array<string, mixed>  $attributes
+     */
+    public static function from(array $attributes): self
+    {
+        return new self(
+            name: (string) $attributes['name'],
```

PHPStan (2.1.17)

Cannot cast mixed to string.


(cast.string)